### PR TITLE
EPMRPP-97895 || Add validation on entered URL

### PR DESCRIPTION
**Pull request by BlazarQSO**
## PR Checklist

* [x] Have you verified that the PR is pointing to the correct target branch? (`develop` for features/bugfixes, other if mentioned in the task)
* [x] Have you verified that your branch is consistent with the target branch and has no conflicts? (if not, make a rebase under the target branch)
* [x] Have you checked that everything works within the branch according to the task description and tested it locally?
* [x] Have you run the linter (`npm run lint`) prior to submission? Enable the git hook on commit in your IDE to run it and format the code automatically.
* [x] Have you run the tests locally and added/updated them if needed?
* [x] Have you checked that app can be built (`npm run build`)?
* [x] Have you checked that no new circular dependencies appreared with your changes? (the webpack plugin reports circular dependencies within the `dev` npm script)
* [x] Have you made sure that all the necessary pipelines has been successfully completed?
* [x] If the task requires translations to be updated, have you done this by running the `manage:translations` script?

### CODE REVIEW

File: app/src/components/integrations/integrationProviders/jiraIntegration/jiraConnectionFormFields/jiraConnectionFormFields.jsx [hunk 1]
```diff
@@ -90,7 +90,7 @@ export class JiraConnectionFormFields extends Component {
         <FieldElement
           name="url"
           label={formatMessage(COMMON_BTS_MESSAGES.linkToBtsLabel)}
-          validate={commonValidators.btsUrl}
+          validate={commonValidators.btsJiraUrl}
```
Comment (AmsterGet) [inline app/src/components/integrations/integrationProviders/jiraIntegration/jiraConnectionFormFields/jiraConnectionFormFields.jsx:93]: This validation should be done for Jira **Cloud**, not Jira Server as the Jira Server can be hosted on any domain.
image
So, please update validation in [Jira Cloud plugin](https://github.com/reportportal/plugin-bts-jira-cloud).

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/components/integrations/integrationProviders/jiraIntegration/jiraConnectionFormFields/jiraConnectionFormFields.jsx (hunk 1 above)
Comment (BlazarQSO) [reply app/src/components/integrations/integrationProviders/jiraIntegration/jiraConnectionFormFields/jiraConnectionFormFields.jsx]: Done.

File: app/src/common/utils/validation/commonValidators.js [hunk 2]
```diff
@@ -37,6 +37,8 @@ export const descriptionEntity = bindMessageToValidator(
 );
 
 export const btsUrl = bindMessageToValidator(validate.url, 'btsUrlHint');
+export const btsJiraCloudUrl = bindMessageToValidator(validate.jiraCloudUrl, 'btsUrlHint');
```
Comment (AmsterGet) [inline app/src/common/utils/validation/commonValidators.js:40]: Pls remove Jira Cloud related changes.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/common/utils/validation/commonValidators.js (hunk 2 above)
Comment (BlazarQSO) [reply app/src/common/utils/validation/commonValidators.js]: Done.

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**